Week-2/InterfaceExample/InterfaceExample.java

```diff
@@ -0,0 +1,12 @@
+public class InterfaceExample {
+
+    public static void main(String[] args) {
+
+       Bike bike= new RoyalEnfield();
+        bike.Company();
+        bike.SpeedLimit();
+        bike = new HarleyDavidson();
+        bike.Company();

```

Comment:
Please give input and compare with  restrict the speed limit of a bike to 90 KMPH and display proper message

Reply:
code is fixed